docs/plugins/graphback-client.md

```diff
@@ -37,9 +37,8 @@ Plugin allows to create files in many formats to suit the needs of your applicat
    *
```

Comment:
We should mention that we generate graphql as single file and this should be used by generators. We should list apollo generator and graphql-code-gen.

Then for non generator approach, we generating typescript to be used directly